Add Remove and Update export functions

MaximusR9 opened the pull request:
**Describe Pull request**
Added remove and update functions to dynamically remove or update jobs/items/gangs without having to restart the server.
It can be useful in scripts where the developer is creating jobs/items/gangs dynamically so for example he created an item but have to modify it so instead of having to restart the whole server he can simply use UpdateItem export.

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest) yes
- Does your code fit the style guidelines? [yes/no] yes
- Does your PR fit the contribution guidelines? [yes/no] yes

GhzGarage commented:
![image](https://user-images.githubusercontent.com/57848836/173965788-b644d5c0-d412-426c-a427-b7abf82a8d1c.png)

MaximusR9 commented:
@GhzGarage looks like someone else added the UpdateItem function. removed my version of the function and removed the unused argument "gang"

Try now should work.
